R/plot2.R

```diff
@@ -256,8 +266,10 @@ plot2.default = function(
     palette = NULL,
     legend = NULL,
     pch = NULL,
-    col = NULL,
     lty = NULL,
+    col = NULL,
+    bg = NA,
```

Comment:
Should default be `NULL` like for the other arguments?

Reply:
Good point. I just took from the `plot.xy` defaults, but we should rather strive to be internally consistent here.